scripts/pyparser/aggregate_reqs.py

```diff
@@ -0,0 +1,99 @@
+import pyparsing as pp
+from pyparsing import oneOf
+import json
+import copy
+
+
+# https://utdallas.collegescheduler.com/api/terms/2021%20Spring/subjects/CS/courses/3345
+
+with open("data/new.json", 'r') as f:
+    data = json.loads(f.read())
+with open("data/major_requirements_processed.txt", 'r') as f:
+    degreePlan = json.loads(f.read())
+with open("data/ecs_computer-science.json", 'r') as f:
+    ecs_cs = json.loads(f.read())
+
+
```

Comment:
Add some doctstrings to your functions to clarify their purposes in natural language.